Sources/Honeycomb/HoneycombDebug.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+internal func debugOptions(options: HoneycombOptions) {
```

Comment:
I used the JS library as my base for debug messaging and I didn't see a lot of places where we printed debug statements aside from the config options.

I can think of a few other places such as the sampler and span processor that we could add statements if we think it's useful. Not sure what would be printed though.

Reply:
I think there's some value in making it consistent with the web SDK, although I am a little curious why the web SDK is so selective about what it prints. I would've expected it to basically just print the options map as-is. 🤔 But that's fine. We can start with this, and if we want to change it, we can change both.

I would suggest renaming the method to `configureDebug` to be even more consistent with the web SDK. I think the name `debugOptions` is kinda confusing.

Also, completely optional, but I think it would be okay to put the 🐝 emoji in the first string like web has, if you want to. Apple is pretty good about emoji support everywhere.